src/edu/csus/ecs/pc2/core/execute/Executable.java

```diff
@@ -211,6 +239,12 @@ public class Executable extends Plugin implements IExecutable {
     private String packageName = "";
 
     private String packagePath = "";
+    
+    //setting this to True will override the prohibition on invoking a Sandbox when running on Windows.
+    // Note that THIS IS FOR DEBUGGING PURPOSES; it does NOT imply any support for Windows sandboxing.
+    //TODO: change this variable to FALSE before generating a production distribution.
+    private boolean debugAllowSandboxInvocationOnWindows = true;
```

Comment:
I recall that I added this during my original development.  Is this still needed in the new development?  If not, it should probably just be removed.  If it's still useful for debugging, then we need to figure out how to ensure that a last-minute step gets executed to set this to false.  I don't think we have anything in our workflow that supports such a thing...

Reply:
At the very least, it should be set to false.  I think I set it to true to test the pc2syscheck.cmd out, and never put it back. *sigh*.